backend/models/volunteer_model.py

```diff
@@ -22,25 +22,25 @@ class Tags(Document):
 class Volunteer(Document):
     first_name = StringField(max_length=500, required=True)
     last_name = StringField(max_length=500, required=True)
-    phone = IntField(min_value=16, max_value=120)
+    phone = StringField(max_length=255)
```

Comment:
max_length for a phone number is 8, and maybe add a regex check to be only digits?

Reply:
Yes, you are right, regex will be added to accept only digits.
Ok, I will set max_length = 8 numbers, but we will not be able to enter mobile phone number with leading zero or full number with country code + city code
Ex.: 
069000000 - 9 numbers
37322000000 - country code + city code + landline phone number
If it will not be a problem, then OK )

Reply:
@johnsmithm If you don't have any questions on this, please resolve it

Reply:
Ok, thank you!